lib/routes/zxcs/novel.ts

```diff
@@ -0,0 +1,112 @@
+import { load } from 'cheerio';
+import cache from '@/utils/cache';
+import ofetch from '@/utils/ofetch';
+import { parseDate } from '@/utils/parse-date';
+import type { Route, DataItem } from '@/types';
+
+export const route: Route = {
+    path: '/novel/:type',
+    name: '小说列表',
+    url: 'zxcs.info',
+    maintainers: ['liaocl'],
+    example: '/zxcs/novel/jinqigengxin',
+    parameters: { type: '小说类型, 可在对应类型页 URL 中找到' },
+    description: `支持小说类型：
+    - jinqigengxin 近期更新
+    - dushi 都市
+    - xianxia 仙侠
+    - xuanhuan 玄幻
+    - qihuan 奇幻
+    - lishi 历史
+    - youxi 游戏
+    - wuxia 武侠
+    - kehuan 科幻
+    - tiyu 体育
+    - lingyi 灵异
+    - junshi 军事
+    - erciyuan 轻小说
+    `,
```

Comment:
```suggestion
- jinqigengxin 近期更新
- dushi 都市
- xianxia 仙侠
- xuanhuan 玄幻
- qihuan 奇幻
- lishi 历史
- youxi 游戏
- wuxia 武侠
- kehuan 科幻
- tiyu 体育
- lingyi 灵异
- junshi 军事
- erciyuan 轻小说
`,
```
Indent the description with either 0 or 2 spaces, or the markdown content won't be rendered properly.